src/core/network.js

```diff
@@ -115,20 +133,17 @@ var NetworkManager = (function NetworkManagerClosure() {
         pendingRequest.expectedStatus = 200;
       }
 
-      if (args.onProgressiveData) {
-        // Some legacy browsers might throw an exception.
-        try {
-          xhr.responseType = 'moz-chunked-arraybuffer';
-        } catch(e) {}
-        if (xhr.responseType === 'moz-chunked-arraybuffer') {
-          pendingRequest.onProgressiveData = args.onProgressiveData;
-          pendingRequest.mozChunked = true;
-        } else {
-          xhr.responseType = 'arraybuffer';
-        }
+//#if (GENERIC || FIREFOX || MOZCENTRAL)
```

Comment:
per comment above, this shall be `#if !(FIREFOX || MOZCENTRAL || CHROMIUM)` and add above

```
#if (FIREFOX || MOZCENTRAL)
xhr.responseType = 'moz-chunked-arraybuffer';
#endif
```

Per https://bugzilla.mozilla.org/show_bug.cgi?id=687087, moz-chunked-arraybuffer was added in FF9 and it's way out of our support range for FF add-on.